### ISSUE TITLE
Misleading transferring output

### ISSUE THREAD

**Issue by vladaionescu** [earthly/earthly#3183]
At the end of the build, when outputting images locally the output says

```transferring docker.io/some/image```

This is misleading because the user might think that the image is being pushed to Docker Hub.

This is not what happens - internally Earthly simply uses a normalized version of the image name, with the registry docker.io, when none is specified. It uses this for the transfer from buildkit to the localhost Docker.

Ideally, this message is clearer, to prevent panicking users who do not intend to publish the image.

**Comment by taytzehao** [comment]
take

**Comment by mikejholly** [comment]
@vladaionescu what's the best way to reproduce this? I don't see the text output in a few test commands. Is this still the case with Logbus/Logstream?

![image](https://github.com/earthly/earthly/assets/332408/66331714-4eb1-423c-abbb-d44023d3a9e8)

This may be where the text is set in BuildKit, but I don't see this being output in Core. https://github.com/earthly/buildkit/blob/earthly-main/source/local/source.go#L244

**Comment by alexcb** [comment]
this last occurred under earthly v0.7.17; (18 was never fully released); then under v0.7.19 this no longer occurred.

under v0.7.19 logbus was feature flipped -- from the changelog:

```
### Changed
- Log sharing will now stream logs as your build is running (rather than uploading logs when build execution completes).
```

We can close this issue.
